plugins/octopus-deploy/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@backstage/plugin-octopus-deploy",
-  "version": "0.1.1-next.1",
+  "version": "0.1.2-next.0",
```

Comment:
Don't need to change this, our workflow will do it automatically when there's a release based off the changesets.